### PR TITLE
remove outdated  linux dependencies

### PR DESCRIPTION
**Pull request by anamnavi**
remove outdated linux dependencies from packaging.psm1 file, specifically liblttng-ust0, libcurl libuuid libuniwnd already removed

remove linux dependency liblttng-ust0. Others mentioned in this issue #14597  were already removed.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Comment by adityapatwardhan** [comment]
Re-running failed test on Windows, seems unrelated to change.

File: .vscode/settings.json [hunk 1]
```diff
@@ -36,5 +36,6 @@
   "powershell.codeFormatting.whitespaceAfterSeparator": true,
 
   // Omnisharp : Enable EditorConfig support
```
Comment (adityapatwardhan) [inline .vscode/settings.json:38]: This file should not be part of the PR.

**Comment by anamnavi** [comment]
Failing test is: "WinRM based remoting session abrupt disconnect"...I think this may be related to Disconnect issue Rob was mentioning last week. Will reach out to him

File: tools/packaging/packaging.psm1 [hunk 2]
```diff
@@ -1268,7 +1268,6 @@ function Get-PackageDependencies
                 "libc6",
                 "libgcc1",
                 "libgssapi-krb5-2",
-                "liblttng-ust0",
```
Comment (adityapatwardhan) [inline tools/packaging/packaging.psm1:1271]: Have you verified with the packages generated after this change can be built and installed?

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (anamnavi) [reply .vscode/settings.json:38]: ah right, I think my editor added that. Let me resolve that :)

File: tools/packaging/packaging.psm1 (hunk 2 above)
Comment (TravisEz13) [reply tools/packaging/packaging.psm1:1271]: Balu verified and asked for this.

**Review by TravisEz13 — APPROVED** [review]

**Review by adityapatwardhan — APPROVED** [review]

File: tools/packaging/packaging.psm1 (hunk 2 above)
Comment (anamnavi) [reply tools/packaging/packaging.psm1:1271]: verified this and ran on Coordinated pipeline and tested artifact deb package.
